addons/l10n_tr_nilvera_einvoice/models/account_edi_xml_ubl_tr.py

```diff
@@ -1,31 +1,5 @@
 from odoo import models
 
-UOM_TO_UNECE_CODE = {
```

Comment:
Stricto senso, it's not stable compliant, but I doubt anybody would have used it already (2 weeks) in a custo, so I agree with the removal.